client/app.html

```diff
@@ -43,7 +44,7 @@
     <script src="services/code_preprocessors/CodePreprocessorDispatcherService.js"></script>
     <script src="services/code_preprocessors/PythonCodePreprocessorService.js"></script>
   </head>
-  <body>
+  <body class="{{bodyClass}}">
```

Comment:
Do you think we could declare the bodyClass on the topmost DOM element inside the learner-view directive instead, rather than on the body tag? It might be necessary to introduce another wrapper div around the whole thing (I'm not sure) but I think that that would be fine if it's needed.